server/src/routes/auth.js

```diff
@@ -38,4 +38,12 @@ export default (app) => {
 
     return res.sendStatus(401);
   });
+
+  app.get('/', (req, res) => {
```

Comment:
routes should be incremental?

Reply:
'/' route is just a placeholder, it'll be removed later

Reply:
Coool, I can handle it!

Reply:
Merged #9